README.md

```diff
@@ -1,5 +1,5 @@
-![MuseScore](mscore/data/musescore_logo_full.png)
- Music notation and composition software
+![MuseScore](mscore/data/musescore_logo_full.png)  
```

Comment:
those 2 trailing spaces are needed? But the leading space harms?

Reply:
The leading space doesn't affect the preview.

Reply:
better drop that leading space though, it wasn't in before and is not needed at all

Reply:
It had actually been there and I just deleted it...

Reply:
Ah, I see. That other PR didn't add it though